clients/venice-admin-tool/src/test/java/com/linkedin/venice/TestDataRecoveryClient.java

```diff
@@ -217,7 +217,7 @@ private void monitorRecovery(ExecutionStatus status, boolean isCurrentVersionNew
     doReturn(storeResponse).when(mockedCli).getStore(anyString());
     doReturn(discoveryResponse).when(mockedCli).discoverCluster(anyString());
     doReturn(statusResponse).when(mockedCli).getFutureVersions(anyString(), anyString());
-    doReturn(queryResponse).when(mockedCli).queryDetailedJobStatus(anyString(), anyString());
+    doReturn(queryResponse).when(mockedCli).queryDetailedJobStatus(any(), anyString());
```

Comment:
The fix itself looks good to me, but I am curious - why it is not failing consistently in this case? Is the check itself also not deterministic in terms of input?

Reply:
That's a good question and I don't know the exact reason, I suspect it might be some optimization done by the JVM compiler. I also did some reproduction experiments myself, e.g. run the same tests 1000 times with/out changing it from anyString -> any, and I did see few failure cases if I didn't change, but no failure if I changed.

Reply:
okay...that's weird, but I guess we could live with it for now :)